### PR TITLE
Create recipe

### PR DESCRIPTION
**Pull request by roxuy**
#### Board:
* [Ticket #5](https://www.notion.so/5-Create-Recipe-1120fb200b4281a9abf5dc2826b9f150?pvs=4)
---
#### Description:
This PR creates Recipes model. 

Screen Shot 2024-10-10 at 23 54 30

---
#### Notes:

---
#### Tasks:
- [x] Add each element in this format
---
#### Risk:
*
---

### CODE REVIEW

File: app/models/recipe.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Indexes
+#
+#  index_recipes_on_user_id  (user_id)
+#
+class Recipe < ApplicationRecord
+  belongs_to :user
+
+  validates :name, presence: true
+  validates :description, presence: true
+  validates :ingredients, presence: true
```
Comment (enzofab91) [inline app/models/recipe.rb]:
```suggestion
  validates :name, :description, :ingredients, presence: true
```

File: spec/factories/recipes.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_recipes_on_user_id  (user_id)
+#
+FactoryBot.define do
+  factory :recipe do
+    name { 'MyString' }
```
Comment (enzofab91) [inline spec/factories/recipes.rb]: What if you use some fake data to ensure randomness in testing? Check https://github.com/faker-ruby/faker

File: spec/models/recipe_spec.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_recipes_on_user_id  (user_id)
+#
+require 'rails_helper'
+
+RSpec.describe Recipe do
```
Comment (enzofab91) [inline spec/models/recipe_spec.rb:21]: You can also test model validations, check https://github.com/thoughtbot/shoulda-matchers?tab=readme-ov-file#activemodel-matchers

**Review by enzofab91 — COMMENTED** [review]
Good work! Left some comments

File: spec/factories/recipes.rb (hunk 2 above)
Comment (roxuy) [reply spec/factories/recipes.rb]: Nice suggestions, i never imagined finding fakers even for food names 😆

**Review by enzofab91 — APPROVED** [review]
:shipit:

**Pull request merged**